### PR TITLE
fix(define): registerComponent no longer requires className as param …

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #133

## Summary:

remove className as a param and always use AuroDatePicker class.

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Review by Izook — DISMISSED** [review]
Yugi Thumb

File: index.js [hunk 1]
```diff
@@ -9,9 +9,9 @@ import { AuroDatePicker } from "./src/auro-datepicker";
  * @param {Function} className - Class name to use for custom element.
```
Comment (jordanjones243) [inline index.js:9]: Remove className as param

Comment (jordanjones243) [inline index.js:9]: Remove className as param

**Review by jordanjones243 — DISMISSED** [review]

**Review by Izook — APPROVED** [review]
LGTM!

**Review by jordanjones243 — APPROVED** [review]

**Pull request merged**